### ISSUE TITLE
The second transport icon is too small

### ISSUE THREAD

**Issue by Altonss** [Flunzmas/seasoncalendar#114]
**Describe the bug**
The second transport icon (when there are 2 side by side) is too small in the main view (in the detailed view of 1 item it is okay!).

**To Reproduce**
1. Go to the main view
2. See an item that has 2 means of transportation

**Expected behavior**
The scaling of the second icon should be like on the item view.

**Screenshots**
![image](https://user-images.githubusercontent.com/66519591/151879307-f918c095-6406-4953-b40d-14874574bf2b.png)
![image](https://user-images.githubusercontent.com/66519591/151879357-06954f55-10a1-44d1-9055-1b8f7f04d80b.png)

**Used device (please complete the following information):**
- Device: Samsung Galaxy Core Prime
- OS: Android 5.0.1
- Browser: Fennec F-Droid
- App Version: 1.4.8+43(f)

**Comment by matzebond** [comment]
Just fund that there is some scaling applied. Looks like the truck icon is supposed to be drawn a little smaller.
https://github.com/seasoncalendar/seasoncalendar/blob/ce5d70aea1ccda3e28197bc25b2cf33054d2e708/lib/components/food_tile.dart#L234-L242
We will need to decide which way to go about this and use the same layout in both cases.

**Comment by Flunzmas** [comment]
Yeah that scaling actually was intentional, to show that it's the "secondary" availability. However, I'm open to alternatives, because I agree that it does look a bit off this way.
